bench/hamming/series.fpcore

```diff
@@ -28,9 +30,11 @@
 
 (FPCore (x)
  :name "qlog (example 3.10)"
- :pre (and (< -1 x) (< x 1))
+ ; We want to sample small x according to Hamming. We can sample more inputs closer to zero by shrinking to +/- 0.5
+ :pre (<= (fabs x) (/ 1 2))
```

Comment:
I think these two are pretty similar except instead of +/- 1 you've chosen +/- 0.5; any reason?

Reply:
@JPepperwood123 you did 2sin correct?

Reply:
Oh sorry. In the textbook Hamming talks about about this example for small x. I interpreted this as sampling points close to 0 so shrinking the range a bit to [-0.5,0.5] would sample more of those points. This is also where the original expression is the least accurate.

Reply:
Why not +-1, which you use in various places?

Reply:
Changed